io/README.md

```diff
@@ -0,0 +1,10 @@
+Rust-Bitcoin IO Library
+=======================
+
+The `std::io` module is not exposed in `no-std` Rust, building `no-std` applications which require
```

Comment:
In b82d54985b4dba33bf1bc312dd1a60ce5a6bb40c:

Should add `so ` before `building`

Reply:
and below